setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
```

Comment:
I guess I'm skeptical that they'll form a cohesive whole, but we can start with this.